### ISSUE TITLE
Remove 'multiple ways' failure from a11y statement

### ISSUE THREAD

**Issue by helennickols** [ministryofjustice/moj-frontend#2438]
## What

Remove the following content from the a11y statement once our a11y statement has confirmed that the sitemap meets WCAG requirements:

> Non-accessible content
> The content listed below is non-accessible for the following reasons.
> 
> Non-compliance with the accessibility regulations
> There’s only one way for users to get to content. This fails WCAG 2.2 Multiple Ways (Level AA). We plan to address this issue by September 2026.

## Why

Ensure that the statement reflects that this problem has been fixed.

## Who needs to be involved

- Content Designer
- Accessibility Specialist

## Done

Content has been removed.

**Comment by helennickols** [comment]
Blocked whilst our a11y specialist comes back to us.

**Comment by helennickols** [comment]
@euhyung Would you mind looking at the [preview](https://moj-frontend-pr-2442.apps.live.cloud-platform.service.justice.gov.uk/accessibility/) for these changes? This will help me to ensure that the changes are correct. I've numbered the changes to make feedback easier:

1. Deleted content:

> (H2) How accessible this website is
> We know some parts of this website are not fully accessible:
> 
> - users cannot get to content in multiple ways

2. Under 'Compliance status' 
Changed from:
> This website is partially compliant with the [Web Content Accessibility Guidelines version 2.2](https://www.w3.org/TR/WCAG22/) AA standard, due to the non-compliances listed below.

to:
> This website is fully compliant with the [Web Content Accessibility Guidelines version 2.2](https://www.w3.org/TR/WCAG22/) AA standard.

Reason: The area of non-compliance has been addressed. 

3. Under 'Preparation of this accessibility statement'
Changed from:
> This statement was prepared on 16 August 2022. It was last reviewed on 23 January 2026.

to:
> This statement was prepared on 16 August 2022. It was last reviewed on 2 July 2026.

Reason: review date reflects when this change has been made.

**Comment by euhyung** [comment]
Hi Helen, thanks for looping me in. 
2 minor things:
You probably want to remove the section under the heading "Non-accessible content" ie the whole block including the heading 
"The content listed below is non-accessible for the following reasons.
Non-compliance with the accessibility regulations
There’s only one way for users to get to content. This fails WCAG 2.2 Multiple Ways (Level AA). We plan to address this issue by September 2026."

The other comment: 
"This website is fully compliant with the [Web Content Accessibility Guidelines version 2.2](https://www.w3.org/TR/WCAG22/) AA standard." 
To make it clearer, could we add a distinction between the platform (this we claim to be fully compliant" rather than the whole website (as we're hosting components in there where we're not 100% on accessibility). 
So even if we say "This website is fully compliant with the [Web Content Accessibility Guidelines version 2.2](https://www.w3.org/TR/WCAG22/) AA standard." could we change it to something more like "The hosting platform / website [...] fully compliant with.[...]"   

WDYT?

**Comment by helennickols** [comment]
@euhyung Thanks for your comments. :-) 

1. The heading in the preview excludes the content under the heading 'Non-accessible content' so that's all done. 

2. Agree that we're not 100% on accessibility for some of the components. We use the term 'website' throughout this statement to refer to the documentation site (design-patterns.service.justice.gov.uk/), rather than frontend and Figma. Do you think the term 'website' could be confused with building blocks themselves?

**Comment by euhyung** [comment]
1 - cool, thanks for clarifying
2 - yes, that's it, thanks - when using 'website' i was wondering if people then might believe all of the included building blocks are then deemed accessible - which is something we'd rather avoid. anything you can think of to make that distinction would be appreciated

**Comment by helennickols** [comment]
@euhyung Thank you! 

2. Do you know if the template allows us to change the following content?

> This accessibility statement applies to the MOJ Design System documentation website.

If so, I'd propose that we change it to:

> This accessibility statement applies to the MOJ Design System documentation website. It does not relate to the components and patterns from the MOJ Frontend codebase which appear in the examples throughout the Design System.

I lifted it from the [GOV.UK Design System a11y statement](https://design-system.service.gov.uk/accessibility-statement/). 

Let me know your thoughts. @chrispymm is this reference accurate?

**Comment by euhyung** [comment]
hey @helennickols!
sounds good to me. and yes, we're allowed to change the some of the content, the [model statement](https://www.gov.uk/guidance/model-accessibility-statement#accessibility-statement) gives you an outline of the required wording ie the words in the brackets below are meant to be adjusted: 
> This accessibility statement applies to [insert scope of statement, for example, website(s)/mobile application(s) to which the statement applies, as appropriate].
